### DISCUSSION TITLE
Status 3 error:  Err_Double_Spend??

### DISCUSSION THREAD

**Discussion by cjcoopertag** [ericaltendorf/plotman#602] [Q&A]
So I am getting the above mentioned warning error in my log?  Does anyone know what that is or better yet how to overcome this?  I have almost 600 plots and no xch and opened up the debug log today to see a mount of this errors.  Also, I have not had a single harvester transaction achieve the 5 seconds or less...the lowest has been 5.1 seconds but that is few in far between all of the mentioned errors in the log.

I have two PC's dedicated to plotting and two that are harvesting.  

I appreciate the input

**Comment by cjcoopertag** [comment]
Well it appears that Mariano54 already commented on this on Reddit.  Said this is normal and can be ignored, should be hidden with next update.  However, I do have a question on harvesting and how is the 5 second or under achieved?  I am running NVME SSD for my temp and WD Enterprise for my final drive.  I have 11 HDD's that are attached to the plot files and I have all four of my machines that are harvesting from the same plots.

**Comment by altendky** [comment]
These are all chia questions, not plotman questions, sorry.

**Reply by cjcoopertag** [reply]
Sorry about that, I forgot I was still on this page.
